### PR TITLE
test(e2e): add wsdl import tests

### PR DESCRIPTION
**Pull request by gcusnieux**
**Issue**

https://github.com/gravitee-io/issues/issues/7295

🚀 CI was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/7295-add-wsdl-import-tests-run-e2e/index.html)
_Notes_: The deployed app is linked to the management API of APIM master. (Same login and password as APIM master)

### CODE REVIEW

**Comment by gcusnieux** [comment]
@gravitee-io/apim 🆙

File: gravitee-apim-e2e/api-test/apis/api-import.spec.ts [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+          definitionVersion,
+          importSwaggerDescriptorEntity,
+        }),
+        201,
+      );
+
+      apiId = api.id;
+      expect(api.id).not.toEqual('');
+      expect(api.visibility).toEqual('PRIVATE');
+      expect(api.state).toEqual('STOPPED');
```
Comment (marcambier) [inline gravitee-apim-e2e/api-test/apis/api-import.spec.ts]:
```suggestion
      expect(api.state).toEqual(ApiEntityStateEnum.STOPPED);
```

File: gravitee-apim-e2e/api-test/apis/api-import.spec.ts [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+          definitionVersion,
+          importSwaggerDescriptorEntity: { ...importSwaggerDescriptorEntity, with_documentation: true },
+        }),
+        201,
+      );
+
+      apiId = api.id;
+      expect(api.id).not.toEqual('');
+      expect(api.visibility).toEqual('PRIVATE');
+      expect(api.state).toEqual('STOPPED');
```
Comment (marcambier) [inline gravitee-apim-e2e/api-test/apis/api-import.spec.ts]:
```suggestion
      expect(api.state).toEqual(ApiEntityStateEnum.STOPPED);
```

File: gravitee-apim-e2e/api-test/apis/api-import.spec.ts [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+          envId,
+          definitionVersion,
+          importSwaggerDescriptorEntity,
+        }),
+        201,
+      );
+
+      apiId = api.id;
+      expect(api.id).not.toEqual('');
+      expect(api.visibility).toEqual('PRIVATE');
```
Comment (marcambier) [inline gravitee-apim-e2e/api-test/apis/api-import.spec.ts]:
```suggestion
      expect(api.visibility).toEqual(Visibility.PRIVATE);
```

File: gravitee-apim-e2e/api-test/apis/api-import.spec.ts [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+          envId,
+          definitionVersion,
+          importSwaggerDescriptorEntity: { ...importSwaggerDescriptorEntity, with_documentation: true },
+        }),
+        201,
+      );
+
+      apiId = api.id;
+      expect(api.id).not.toEqual('');
+      expect(api.visibility).toEqual('PRIVATE');
```
Comment (marcambier) [inline gravitee-apim-e2e/api-test/apis/api-import.spec.ts]:
```suggestion
      expect(api.visibility).toEqual(Visibility.PRIVATE);
```

**Review by marcambier — APPROVED** [review]

**Pull request merged**